lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_GeneratePercentilesFromProbabilities.py

```diff
@@ -536,15 +536,15 @@ def test_check_data_specifying_no_of_percentiles(self):
         Test that the plugin returns an Iris.cube.Cube with the expected
         data values for a specific number of percentiles.
         """
-        data = np.array([[[[21.5, 8.75, 11.],
-                           [8.33333333, 8.75, -16.],
-                           [-16., -25., -28.]]],
-                         [[[31., 10., 12.],
+        data = np.array([[[[24., 8.75, 11.],
+                           [8.33333333, 8.75, -66.],
+                           [-66., -93.75, -103.]]],
+                         [[[36., 10., 12.],
                            [10., 10., 8.],
-                           [8., -10., -16.]]],
-                         [[[40.5, 11.66666667, 31.],
+                           [8., -47.5, -66.]]],
+                         [[[48., 11.66666667, 36.],
                            [11.66666667, 11., 10.5],
-                           [9.66666667, 5., -4.]]]])
+                           [9.66666667, -1.25, -29.]]]])
```

Comment:
These change because of the change in the ECC bounds? I guess when we have more thresholds these are less important, but they still have a surprising impact, which I guess we would still see in the tails of the distribution?

Reply:
It is only the points at the tails that have changed, and I went through and checked the numbers. It is because you only have a few thresholds in this test and they are relatively close together compared to the bounds we now have.